hubspot3/tickets.py

```diff
@@ -78,3 +90,171 @@ def get_all(self, limit: int = -1, **options) -> list:
             offset = batch["offset"]
 
         return output if not limited else output[:limit]
+
+    default_batch_properties = [
```

Comment:
You can declare this list as `default_batch_properties = ["subject"]`.

Reply:
This is private?  Fields like this should't be on top of the file?

Reply:
Its is not private, I put it there following contacts.py, but I'm going to put it on top